modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -38,14 +54,28 @@ metadata:
 spec:
   targetNamespaces:
   - openshift-sandboxed-containers-operator
----
+----
+
+.. Create the `OperatorGroup` object:
++
+[source,terminal]
+----
+$ oc create -f <file name>.yaml
+----
+
+. Create the `Subscription` object to subscribe the `Namespace` to the {sandboxed-containers-operator}.
+
+.. Create a `Subscription` object YAML file that contains the following manifest:
++
+[source,yaml,subs="attributes+"]
+----
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: openshift-sandboxed-containers-operator
   namespace: openshift-sandboxed-containers-operator
 spec:
-  channel: "preview-1.1"
+  channel: "preview-{sandboxed-containers-version}"
```

Comment:
This gets rendered as 1.1, will need to be 1.2 for GA

Reply:
Yes. For now this update is going into 4.9. I'll update it separately for 4.10.